libs/langchain-community/src/vectorstores/opensearch.ts

```diff
@@ -27,6 +27,7 @@ interface VectorSearchOptions {
  */
 export interface OpenSearchClientArgs {
   readonly client: Client;
+  readonly service?: 'es' | 'aoss';
```

Comment:
The service name is adapted from the opensearch-js client library.

https://github.com/opensearch-project/opensearch-js/blob/835ad4ec11e03ec3b0293612226f91b859f2553d/lib/aws/index.d.ts#L22